code/modules/virus2/effect.dm

```diff
@@ -377,6 +377,37 @@
 	speech.message = message
 
 
+/datum/disease2/effect/vitreous
+	name = "Vitreous resonance"
+	stage = 2
+	chance = 25
+	max_chance = 75	
+	max_multiplier = 2
+
+datum/disease2/effect/vitreous/activate(var/mob/living/carbon/mob)
+	if(istype(mob, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = mob
+		var/hand_to_use = pick(GRASP_RIGHT_HAND, GRASP_LEFT_HAND)
```

Comment:
Might want to make this work with n amount of hands.

Probably a way to see the max amount of hands a mob has and then go by that with a `rand()`